auth0/resource_auth0_client.go

```diff
@@ -919,15 +947,103 @@ func flattenClientJwtConfiguration(jwt *management.ClientJWTConfiguration) []int
 }
 
 func flattenClientRefreshTokenConfiguration(refreshToken *management.ClientRefreshToken) []interface{} {
+	if refreshToken == nil {
+		return nil
+	}
```

Comment:
We return early in case `refreshToken` is nil. This decreases the level of indentation within this func.